modules/reactor/src/meshgenerators/PatternedHexMeshGenerator.C

```diff
@@ -120,6 +120,11 @@ PatternedHexMeshGenerator::validParams()
       "interface_boundary_id_shift_pattern",
       "User-defined shift values for each pattern cell. A double-indexed array starting with the "
       "upper-left corner.");
+  MooseEnum quad_elem_type("QUAD4 QUAD8 QUAD9", "QUAD4");
+  params.addParam<MooseEnum>(
+      "boundary_region_element_type",
```

Comment:
let's add this to the external boundary parameter group (see existing addParamNamesToGroup below)